src/GovUk.Education.ExploreEducationStatistics.Data.Seed/Services/SeedService.cs

```diff
@@ -31,44 +41,140 @@ public void Seed()
+                                    {
+                                        Id = publication.Id,
+                                        Slug = publication.Slug,
+                                        Title = publication.Title,
+                                        TopicId = topic.Id,
+                                        Topic = new Topic
+                                        {
+                                           Id = topic.Id,
+                                           Slug = topic.Slug,
+                                           Title = topic.Title,
+                                           ThemeId = theme.Id,
+                                           Theme = new Theme
+                                           {
+                                              Id = theme.Id,
+                                              Slug = theme.Slug,
+                                              Title = theme.Title
+                                           }
+                                        }
+                                    },
+                                    Subjects = new []
+                                    {
+                                        new Subject {
+                                            Id = subject.Id,
+                                            Name = subject.Name,
+                                            ReleaseId = release.Id
+                                        }
+                                    }
+                                };
 
-                    var subjects = updated.Topics
-                        .SelectMany(topic => topic.Publications)
-                        .SelectMany(publication => publication.Releases)
-                        .SelectMany(release => release.Subjects);
+                                var file = SamplePublications.SubjectFiles.GetValueOrDefault(subject.Id);
 
-                    Seed(subjects);
+                                StoreFiles(r, file, subject.Name);
+                                Seed(file.ToString() + ".csv", r);
+                            }   
+                        }
+                    } 
                 }
             }
-            finally
-            {
-                _context.ChangeTracker.AutoDetectChangesEnabled = true;
-            }
 
             stopWatch.Stop();
             _logger.LogInformation("Seeding completed with duration {duration} ", stopWatch.Elapsed.ToString());
         }
 
-        private void Seed(IEnumerable<Subject> subjects)
+        private void Seed(string dataFileName, Release release)
         {
-            foreach (var subject in subjects)
+            var storageAccount = CloudStorageAccount.Parse(_storageConnectionString);
+            var client = storageAccount.CreateCloudQueueClient();
+            var pQueue = client.GetQueueReference("imports-pending");
+            var aQueue = client.GetQueueReference("imports-available");
```

Comment:
Similar to be before, I wonder if this aQueue could be created by the Processor function.

Reply:
No because the functions need it to be present already - so yeah could perhaps move to template.

Reply:
Just thought, if it's moved to the template then it won't exist for us locally which would be a problem.